test/build_payment.js

```diff
@@ -7,6 +7,7 @@ describe('Ripple REST Client buildPayment', function() {
 
   before(function () {
     client = new Client({
+      api: 'http://localhost:5990/',
```

Comment:
Oh you're looking at an older version of the PR. Have fixed that. @wltsmrz

On Wednesday, March 11, 2015, wltsmrz <notifications@github.com> wrote:

> In test/build_payment.js
> <https://github.com/ripple/ripple-rest-client/pull/1#discussion_r26258157>
> :
>
> > @@ -7,6 +7,7 @@ describe('Ripple REST Client buildPayment', function() {
> >
> >    before(function () {
> >      client = new Client({
> > +      api: 'http://localhost:5990/',
>
> In a test? Why?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ripple/ripple-rest-client/pull/1/files#r26258157>.
>


-- 
*Abiy Seifu |* Software Engineer
*Ripple Labs, Inc.*
abiy@ripple.com | www.ripple.com